README.md

```diff
@@ -7,36 +7,39 @@ October instance.
 
 ### Dependencies
 
-- PHP 5 (version used in production)
-- [October](https://octobercms.com/)
-- Some database (MySQL is recommended)
+- [Docker](https://docker.com)
```

Comment:
`docker-compose` is also needed (not installed automatically at least when installing `docker` on my distro).